queries/models/bilhetagem_interno/schema.yml

```
@@ -1,4 +1,11 @@
 models:
+  - name: disparos_whatsapp_caso_1_2
+    description: ""
+    columns:
+      - name: vars
+        description: ""
+        policy_tags:
+            - 'projects/rj-smtr-dev/locations/us/taxonomies/1688927184371506251/policyTags/8698618633874150807'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)